indico/modules/events/registration/templates/display/event_header.html

```diff
@@ -55,7 +55,9 @@
             <div class="participant-list-wrapper">
                 <ul class="participant-list">
                     {%- for participant in participants -%}
-                        <li>{{ participant.full_name }}</li>
+                    <li {% if participant.checked_in and participant.registration_form.publish_checkin_enabled %}class="checked-in"{% endif %}>
```

Comment:
yeah either way is fine, but if you had more than one conditional class it'd make more sense for sure (empty `class=""` is not really a problem)